### DISCUSSION TITLE
выбор аккаунта из ASF

### DISCUSSION THREAD

**Discussion by BLYFIN** [Luois45/claim-free-steam-packages#20] [Ideas]
Сделайте пожалуйста выбор аккаунта из ASF для добавления игр, либо возможность добавлять на все аккаунты как через команду !addlicense **asf** app// 
Спасибо!
Извиняюсь за русский.

Please make a choice of an account from ASF to add games, or the ability to add to all accounts as through a team !addlicense **asf** app// 
Thanks!
I apologize for the Russian.

**Comment by louisa-uno** [comment]
Thanks for the feature request.
I've opened the Issue #21 for it and will try to implement it.
